scaleway/helpers.go

```diff
@@ -83,199 +81,11 @@ func expandZonedID(id interface{}) ZonedID {
 	return zonedID
 }
 
-// Bool returns a pointer to of the bool value passed in.
-func Bool(val bool) *bool {
-	return &val
-}
-
 // String returns a pointer to of the string value passed in.
 func String(val string) *string {
```

Comment:
nit: all leftovers of this function should switch to `scw.StringPtr()` (scaleway-sdk-go)

Reply:
https://github.com/scaleway/terraform-provider-scaleway/pull/593